### ISSUE TITLE
recommendations for passing extra arguments through `...` in `lapply() `and al.

### ISSUE THREAD

**Issue by ThomasSoeiro** [tidyverse/style#255]
Should the style guide provide recommendations for passing extra arguments through `...` in `lapply() `and al.? In particular, should the extra arguments be named when passed to a function with no default arguments?

E.g. with function signature `f(x, y)`:

```r
# bad?
lapply(1:3, f, 4)

# good?
lapply(1:3, f, y = 4)
```

**Comment by ThomasSoeiro** [comment]
`lintr::lint("lapply(1:3, f, 4)\n")` (version 2.0.1) does not complain and [Advanced R](https://adv-r.hadley.nz/functions.html#prefix-transform) mentions:

```r
add <- function(x, y) x + y
lapply(list(1:3, 4:5), add, 3)
```

Suggesting that maybe "good" and "bad" are respectively:

```r
# good?
lapply(1:3, f, 4)

# bad?
lapply(1:3, f, y = 4)
```
